imports/api/rest/put-pending-invitations.js

```diff
@@ -0,0 +1,43 @@
+import { PendingInvitations } from '../pending-invitations'
+import { invite } from '../../util/email-invite'
+import { Meteor } from 'meteor/meteor'
+
+export default (req, res) => {
+  if (req.query.accessToken !== process.env.API_ACCESS_TOKEN) {
+    res.send(401)
+    return
+  }
+
+  console.log('Marking done:', req.body)
+  try {
+    const results = PendingInvitations.update({
+      _id: { $in: req.body },
+      done: { $ne: true }
+    },
+      { $set: { done: true } })
+    if (results < 1) {
+      // Nothing happened, so don't send any emails
+      res.send(200, results)
+      return
+    }
```

Comment:
Can you update Meteor.users for the user's invitedToCases attribute (with the right sub-document) with "done: true"?

Reply:
Btw why doesn't invitedToCases reference who invited the user? Shouldn't it?

Reply:
Need to answer the question: "How many ppl has this user invited in the last week?" for https://github.com/unee-t/frontend/issues/116